distribution/src/main/resources/bin/elasticsearch-service.bat

```diff
@@ -4,20 +4,20 @@ SETLOCAL enabledelayedexpansion
 TITLE Elasticsearch Service ${project.version}
 
 IF DEFINED JAVA_HOME (
-  SET JAVA=%JAVA_HOME%\bin\java.exe
+  SET JAVA="%JAVA_HOME%"\bin\java.exe
```

Comment:
Why? I tested it with this change. Do you have an example where this would break without your suggestion?